### ISSUE TITLE
About Eq.3

### ISSUE THREAD

**Issue by JieyuZ2** [varsha33/LCL_loss#6]
Hey,

I think if you really meant to place w outside of log in Eq.3, then in [this line](https://github.com/varsha33/LCL_loss/blob/master/loss.py#L144) it should time logits_mask instead of weighted_mask, right?

Thanks

**Comment by varsha33** [comment]
Only for the positive numerator weights, wi yi as mentioned in the note is outside the log. This line takes care of the denominator.

**Comment by JieyuZ2** [comment]
gotcha!
